cg/meta/workflow/mip_rna.py

```diff
@@ -54,8 +54,10 @@ def config_sample(self, link_obj, panel_bed: str | None = None) -> dict[str, str
             sample_data[Pedigree.FATHER.value]: str = link_obj.father.internal_id
         return sample_data
 
-    def panel(self, case_id: str, genome_build: str = GENOME_BUILD_38) -> list[str]:
-        """Create the aggregated gene panel file"""
-        case_obj: Case = self.status_db.get_case_by_internal_id(internal_id=case_id)
-        all_panels = self.convert_panels(case_obj.customer.internal_id, case_obj.panels)
+    def get_gene_panel(self, case_id: str, genome_build: str = GENOME_BUILD_38) -> list[str]:
```

Comment:
could this be a function outside of the classes? It seems to be repeated in mip-rna and mip-dna.

Reply:
Fixed